packages/blockchain-wallet-v4-frontend/src/modals/Exchange/IdentityVerification/Personal/selectors.js

```diff
@@ -59,6 +60,14 @@ export const getData = state => {
     initialCountryCode: selectors.core.settings
       .getCountryCode(state)
       .getOrElse(null),
+    initialEmail: selectors.core.settings.getEmail(state).getOrElse(null),
+    isEmailVerified: () =>
+      selectors.core.settings.getEmailVerified(state).getOrElse(false),
+    emailVerified: selectors.core.settings
+      .getEmailVerified(state)
+      .getOrElse(false),
```

Comment:
isEmailVerified and emailVerified are the same

Reply:
isEmailVerified is a function to determine if email has been already verified at the start of kyc process( called at cdm)
We are hiding email field in this case

Reply:
Got it